### DISCUSSION TITLE
What is "dot" the makefile is looking for?

### DISCUSSION THREAD

**Discussion by JohnDarkhorse** [bewuethr/pandoc-bash-blog#117] [Q&A] [answered]
In the makefile . . .

> $(call checkdep,dot,dot)

I have no clue what that is, and nothing named "dot" can be found at packages.ubuntu.com.  When I search, all I get are endless results explaining "dot files".

Can you provide a link to this software's source code or explain how to get it?

TIA

**Comment by bewuethr** [comment, marked as answer]
Dot is part of [Graphviz](https://graphviz.org/); it generates graphs from textual input (see [this article](https://benjaminwuethrich.dev/2020-06-29-pbb-dot-graphs.html)). It's provided by the `graphviz` package in Ubuntu, and can be installed with

```sh
sudo apt install graphviz
```
